.github/workflows/android-qa-tests.yml

```diff
@@ -0,0 +1,27 @@
+name: Android QA tests
+
+on: workflow_dispatch
+
+jobs:
+  android_qa_tests:
+    runs-on: [macos-latest]
+    steps:
+      - name: Checkout simple-android
+        uses: actions/checkout@v2
+        with:
+          repository: simpledotorg/simple-android
+          path: simple-android
+
+      - name: Setup JDK
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+
+      - name: Turn on capturing of flaky tests
+        run: cat simple-android/app/src/androidTest/resources/quarantine_ci.properties > simple-android/app/src/androidTest/resources/quarantine.properties
```

Comment:
We don't need to capture flaky android tests, methinks.

Reply:
I copied it over from the [android CI checks](https://github.com/simpledotorg/simple-android/blob/master/.github/workflows/ci_checks.yml#L66), the assumption being these are some failing cases we shouldn't care about.

Reply:
Ah, these _rescue_ flaky tests. I thought they capture them for reporting purposes when they fail. Let's confirm with the android team. If it rescues flakers, then yes, leave it in 👍

Reply:
This rescues flaky tests in addition to reporting flaky tests at https://flakiness.dx.obvious.in/simple-android-instrumented-tests/reports

Note: seems like the SSL certificate has expired, which is weird since I used Certbot which _should_ automatically renew them.